packages/clerk-js/src/ui/elements/Pagination.tsx

```diff
@@ -52,32 +53,27 @@ const RowInformation = (props: RowInfoProps) => {
   } = props;
 
   return (
-    <Text>
+    <Text elementDescriptor={descriptors.paginationRowText}>
       <Text
         as='span'
-        sx={t => ({
-          color: t.colors.$blackAlpha700,
-        })}
+        colorScheme='inherit'
         localizationKey={localizationKeys('paginationRowText__displaying')}
       />{' '}
       <Text
         as='span'
+        colorScheme='inherit'
```

Comment:
@octoper I see we made some minor UI changes here, could you please add a screenshot in the PR description if needed? Even if the changes are minor, this always helps and we can use it for future reference as well.

Reply:
Yes, let's add a before and after showcasing why we need to use `inherit` here.

Reply:
The main reason I used `colorScheme='inherit'` here is just to have the colour to the parent element so I can use the element descriptor only theme.

Reply:
@octoper I'm very skeptic about the result of the before/active. This is something i recently touched as well, but endup not making the change.

I don't think using the primary colour here makes sense, and the result is not that elegant.

Reply:
@panteliselef Yeah it's not that elegant but I thought that this should match the other parts of our components that have text, because if this is not changed every other part will have the primary colour and this will be black.